utils/python/CIME/preview_namelists.py

```diff
@@ -76,12 +83,14 @@ def create_namelists(case):
             if 'python' in first_line:
                 expect(False, detail)
             else:
-                run_cmd_no_fail("%s %s" % (cmd, caseroot), verbose=True)
+                run_cmd_no_fail("%s %s" % (cmd, caseroot), verbose=False)
```

Comment:
Should we add the same code here as we did for buildnml to call these as subroutines if possible?